packages/blocks/src/_common/embed-block-helper/embed-block-element.ts

```diff
@@ -81,7 +81,7 @@ export class EmbedBlockElement<
       const blockComponent = anchorComponent as this;
       const isInSurface = blockComponent.isInSurface;
       if (!isInSurface) {
-        this.host.selection.setGroup('block', [
+        this.host.selection.set([
```

Comment:
Why we don't use `setGroup` here?

Reply:
we need to clear other selections to prevent side effects... while dragging only the dragging elements should be selected.
this was changed to solve a dragging bug... where existing text-selection was causing drag/drop to fail.

Reply:
Oh, I suddenly realized that none of the selections belong to the 'block' group, only 'note' and 'edgeless'. Why was it written this way before?

<img width="378" alt="image" src="https://github.com/toeverything/blocksuite/assets/50035259/71b715ff-f6cd-435c-b55b-4769a326730b">

Reply:
oops... that was the reason for the bug then... corrected to `setGroup('note'`